build/generate-manifests.sh

```diff
@@ -56,13 +56,20 @@ function generateCRDs() {
   mv deploy/crds/maistra.io_servicemeshcontrolplanes.yaml ${BUNDLE_DIR}/servicemeshcontrolplanes.crd.yaml
   mv deploy/crds/maistra.io_servicemeshmemberrolls.yaml ${BUNDLE_DIR}/servicemeshmemberrolls.crd.yaml
   mv deploy/crds/maistra.io_servicemeshmembers.yaml ${BUNDLE_DIR}/servicemeshmembers.crd.yaml
+
   rm -rf deploy/crds/
 
   cat ${BUNDLE_DIR}/servicemeshcontrolplanes.crd.yaml >deploy/src/crd.yaml
   echo -e "\n---\n" >>deploy/src/crd.yaml
   cat ${BUNDLE_DIR}/servicemeshmemberrolls.crd.yaml >>deploy/src/crd.yaml
   echo -e "\n---\n" >>deploy/src/crd.yaml
   cat ${BUNDLE_DIR}/servicemeshmembers.crd.yaml >>deploy/src/crd.yaml
+
+  currentDir=$(pwd)
+  cd ${BUNDLE_DIR}
+  sed -i -e '/x-kubernetes-list-map-keys:/,/x-kubernetes-list-type: map/ { /- protocol/d }' *.crd.yaml        
```

Comment:
Can you move this to the original fix, which was on line 53?

Reply:
Sure. I thought that was a separate fix since this doesn't modify default values or descriptions.

Reply:
It fixes the same problem, but it only works on 4.5+.  Having the comment too is a hint about what's going on so we can remove it once we're no longer supporting 4.4.